javascript/javascript1/week4/script.js

```diff
@@ -124,6 +125,24 @@ function getReply(command) {
                     }, minutes * 60 * 1000); // minutes * seconds * milliseconds
 
                     return `Timer set for ${minutes} minutes`;
+
+                case "birthdayMatch":
+                    // Extracting day, month, and year from the 'match' array and creating a new Date object representing the birth date
+                    const birthDate = new Date(
+                        `${match[1]}/${match[2]}/${match[3]}`
+                    ); // dd/mm/yyyy
```

Comment:
I would declare these as their own variables for eligibility

Reply:
sure 👍